wire-ios-sync-engine/Source/SessionManager/SessionManager+AnalyticsUseCases.swift

```diff
@@ -16,26 +16,39 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
+import WireAnalytics
+
 extension SessionManager {
 
     enum AnalyticsSessionError: Error {
 
         case analyticsNotAvailable
+        case analyticsConfigurationNotAvailable
         case missingAnalyticsUserProfile
         case missingActiveUserSession
 
     }
 
     public func makeDisableAnalyticsUseCase() throws -> DisableAnalyticsUseCaseProtocol {
-        guard let analyticsManager else {
-            throw AnalyticsSessionError.analyticsNotAvailable
+
+        guard let userSession = self.activeUserSession else {
+            throw AnalyticsSessionError.missingActiveUserSession
+        }
+
+        let didDisableAnalyticsClosure: () -> Void = {
```

Comment:
Where do we use `didDisableAnalyticsClosure`? I didn't find in the code

Reply:
It was left-over, deleted now.